components/side-drawer/src/vwc-side-drawer.scss

```diff
@@ -75,10 +88,12 @@ $inline-size: var(#{variables.$side-drawer-inline-size}, 280px);
 
 .side-drawer--modal {
 	position: fixed;
-	top: 0;
```

Comment:
I think the top & bottom are still needed

Reply:
@rachelbt Are you sure? It seems to work fine without it.

Reply:
the change was  from right to left. 
The top & bottom is to scratch the side drawer to top + bottom of the browser